source/_components/homekit.markdown

```diff
@@ -113,6 +113,11 @@ homekit:
                 description: Name of the entity to show in HomeKit. HomeKit will cache the name on the first run so a device must be removed and then re-added for any change to take effect.
                 required: false
                 type: string
+              linked_battery_sensor:
+                description: The `entity_id` of the battery entity for the accessory.
```

Comment:
This should mention that it must be a `sensor` entity.

Reply:
As for your comment: https://github.com/home-assistant/home-assistant/pull/22788#issuecomment-481055199
I think the `description` of the `name` attribute above could fit quite well (slightly altered of course).

Nevertheless, what do you think about adding a new topic inside the troubleshooting section. Something like `The linked battery sensor isn't recognized`?

Reply:
I put something together for this in the troubleshooting section, as well as mentioning it in the description. Let me know what you think.